VocaDbWeb/tsconfig.json

```diff
@@ -18,7 +18,8 @@
 			"@Repositories/*": ["Scripts/Repositories/*"],
 			"@Shared/*": ["Scripts/Shared/*"],
 			"@ViewModels/*": ["Scripts/ViewModels/*"]
-		}
+		},
+		"resolveJsonModule": true

```

Comment:
`resolveJsonModule` needs to be set to `true`, so that `tsconfig.json` can be imported from `jest.config.ts`.